lib/ExpensiMark.js

```diff
@@ -766,8 +766,11 @@ export default class ExpensiMark {
      */
     extractLinksInMarkdownComment(comment) {
         try {
-            const html = this.replace(comment);
-            const matches = [...html.matchAll(new RegExp(`<a href="${MARKDOWN_URL_REGEX}" target="_blank" rel="noreferrer noopener">`, 'gi'))];
+            const htmlString = this.replace(comment);
+
+            // We use same anchore tag template as link and autolink rules to extract link
```

Comment:
@eh2077 Can you please fix the typo so that `anchore` becomes `anchor`? Thanks

Reply:
@lakchote Updated

Reply:
Thanks!